lib/amazon/parser/vm.rb

```diff
@@ -22,6 +23,27 @@ def parse_vms(instance, _scope)
 
       private
 
+      def parse_network(instance)
+        network = {
+          :fqdn                 => instance.public_dns_name,
+          :private_ip_address   => instance.private_ip_address,
+          :public_ip_address    => instance.public_ip_address,
+          :mac_addresses        => [],
+          :private_ip_addresses => [],
+          :public_ip_addresses  => [],
+        }
+
+        (instance&.network_interfaces || []).each do |interface|
```

Comment:
Is the safe navigator needed here?  If instance were nil it would have blown up already and `&.` doesn't handle if instance doesn't respond to `.network_interfaces`

Reply:
hm, you're probably right, I'll change it